src/main/java/sparksoniq/jsoniq/runtime/iterator/RuntimeIterator.java

```diff
@@ -134,7 +137,7 @@ public void reset(DynamicContext context) {
     public void write(Kryo kryo, Output output) {
         output.writeBoolean(_hasNext);
```

Comment:
Writing _hasNext, _isOpen and _currentDynamicContext can be completely dropped (s. email).

Reply:
Done